tasks/burp_suite_pro.yml

```diff
@@ -0,0 +1,115 @@
+          # OK [o, Enter], Cancel [c]
+          #
+          # Click Next to continue, or Cancel to exit Setup.
+          # Select the folder where you would like Burp Suite Professional to be
+          # installed, then click Next.
+          # Where should Burp Suite Professional be installed?
+          # [/usr/local/BurpSuitePro]
+          #
+          # Create symlinks?
+          # Yes [y, Enter], No [n]
+          #
+          # Select the folder where you would like Burp Suite Professional to create symlinks, then click Next.
+          # [/usr/local/bin]
+          #
+          # Extracting files ...
+          # Setup has finished installing Burp Suite Professional on your computer.
+          # Finishing installation ...
+          "This will install Burp Suite Professional on your computer\\.": o
+          "Where should Burp Suite Professional be installed\\?": "{{ burp_suite_install_directory }}"
+          "Create symlinks\\?": "y"
+          "Select the folder where you would like Burp Suite Professional to create symlinks, then click Next": "{{ burp_suite_symlinks_directory }}"
+      # This failed_when clause is required because the installer
+      # command returns a nonzero return code even when it succeeds.
+      failed_when:
+        - install_command.rc != 0
+        - install_command.rc != 1
+      register: install_command
+
+    - name: License Burp Suite Pro
+      expect:
+        chdir: "{{ burp_suite_install_directory }}"
+        command: ./jre/bin/java -Djava.awt.headless=true -jar ./burpsuite_pro.jar
+        responses:
+          # $ ./jre/bin/java -Djava.awt.headless=true -jar ./burpsuite_pro.jar
+          # Burp Suite Professional Terms & Conditions of Supply
+          #
+          # <snip>
+          # License text
+          # </snip>
+          # Do you accept the license agreement? (y/n)
+          # y
+          # This version of Burp requires a license key. To continue, please paste your license key below.
+          # <license key>
+          # Burp will now attempt to contact the license server and activate your license. This will require Internet access.
+          # NOTE: license activations are monitored. If you perform too many activations, further activations for this license may be prevented.
+          # Enter preferred activation method (o=online activation; m=manual activation; r=re-enter license key)
+          # o
+          # Your license is successfully installed and activated.
+          "Do you accept the license agreement\\?": "y"
+          "To continue, please paste your license key below\\.": "{{ lookup('file', '/tmp/' + burp_suite_license_object_name) }}"
+          "Enter preferred activation method": o
+      # This failed_when clause is required because the licensing
+      # command returns a nonzero return code even when it succeeds.
+      failed_when:
+        - license_command.rc != 0
+        - license_command.rc != 13
+        - license_command.rc != 129
```

Comment:
* According to `/usr/include/asm-generic/errno-base.h` (which is included indirectly by `/usr/include/errno.h`), an error code of `13` indicates `EACCES` AKA `Access denied`.
* According to `/usr/include/asm-generic/errno.h` (which is included indirectly by `/usr/include/errno.h`), an error code of `129` indicates `EKEYREJECTED` AKA `Key was rejected by service`.  This may be because the Burp Suite servers are refusing to let us use the key a second time.